### PR TITLE
wth - Refresh SPARC Protocol Information if associated to a validated…

### PR DESCRIPTION
**Pull request by wtholt**
… Research Master ID

Wrote a task that updates SPARC Protocol Short Title and Title with the validated Research Master Short Title and Title. Display visual cue to show that this Protocol has been updated with data from the RMID app.
[#148336773]

Story - https://www.pivotaltracker.com/story/show/148336773

### CODE REVIEW

File: app/views/dashboard/protocols/_protocols_list.html.haml [hunk 1]
```diff
@@ -73,6 +73,7 @@
             = protocol.id
         %td.title
           = short_title_display protocol
+          = display_rmid_validated_protocol(protocol, "Short Title")
```
Comment (kyle-glick) [inline app/views/dashboard/protocols/_protocols_list.html.haml]: Can you put the various instances of "Short Title" and "Title" into YAML?

**Comment by Stuart-Johnson** [comment]
@wtholt Can you add the appropriate tags to this PR? Thanks.

File: app/views/dashboard/protocols/_protocols_list.html.haml (hunk 1 above)
Comment (wtholt) [inline app/views/dashboard/protocols/_protocols_list.html.haml]: Sure thing

**Comment by wtholt** [comment]
@Stuart-Johnson I added musc_contribution. What version is next?

**Pull request closed without merging**